sigle/src/modules/home/components/StoryItem.tsx

```diff
@@ -97,81 +105,222 @@ export const StoryItem = ({
-                    Feature this story
-                  </DropdownMenuItem>
+                    Draft
+                  </Box>
                 )}
-                {story.featured && type === 'public' && (
-                  <DropdownMenuItem onSelect={onUnFeature}>
-                    Un-feature this story
-                  </DropdownMenuItem>
+                {user && (
+                  <DropdownMenu>
+                    <DropdownMenuTrigger asChild>
+                      <IconButton
+                        css={{ alignSelf: 'start', p: '$1' }}
+                        aria-label="Story settings"
+                      >
+                        <DotsHorizontalIcon width={22} height={22} />
+                      </IconButton>
+                    </DropdownMenuTrigger>
+                    <DropdownMenuContent>
+                      {type === 'public' && (
+                        <DropdownMenuItem
+                          as="a"
+                          href={`/${user.username}/${story.id}`}
+                          target="_blank"
+                          rel="noopener noreferrer"
+                        >
+                          View my story
+                        </DropdownMenuItem>
+                      )}
+                      <DropdownMenuItem onSelect={onEdit}>
+                        Edit
+                      </DropdownMenuItem>
+                      {!story.featured && type === 'public' && (
+                        <DropdownMenuItem onSelect={onFeature}>
+                          Feature this story
+                        </DropdownMenuItem>
+                      )}
+                      {story.featured && type === 'public' && (
+                        <DropdownMenuItem onSelect={onUnFeature}>
+                          Un-feature this story
+                        </DropdownMenuItem>
+                      )}
+                      <DropdownMenuSeparator />
+                      <DropdownMenuItem color="red" onSelect={onDelete}>
+                        Delete
+                      </DropdownMenuItem>
+                    </DropdownMenuContent>
+                  </DropdownMenu>
                 )}
-                <DropdownMenuSeparator />
-                <DropdownMenuItem color="red" onSelect={onDelete}>
-                  Delete
-                </DropdownMenuItem>
-              </DropdownMenuContent>
-            </DropdownMenu>
-            {story.featured && <StarIcon width={22} height={22} />}
-          </StoryTitleContainer>
-          <StoryDate>
+              </Flex>
+            </Flex>
             <Link
               href="/stories/[storyId]"
               as={`/stories/${story.id}`}
               passHref
             >
-              <a>{format(story.createdAt, 'HH:mm dd MMMM yyyy')}</a>
+              <Text size="xs" as="a">
+                {format(story.createdAt, 'MMMM dd, yyyy ')}
+                at
+                {format(story.createdAt, ' h:mmaaa')}
```

Comment:
using `h` is not prefixed with 0 for numbers < 10, but using `mm` does prefix it. To be consistent let's decide on prefixing both or not prefixing both :)

Reply:
ah I see, so would it make sense to do something like `h:maaa` for not prefixed or  `hh:mmaaa` for prefixed ?

Reply:
exactly 👍